Dockerfile

```diff
@@ -9,13 +9,16 @@
 #
 # Description: Docker container image recipe
 
-FROM debian:jessie
+FROM debian:jessie-slim
 
 LABEL maintainer="Fossology <fossology@fossology.org>"
 
 WORKDIR /fossology
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+# Fix for slim images
+RUN mkdir /usr/share/man/man1 \
+ && mkdir /usr/share/man/man7 \
```

Comment:
It is not obvious why we need this fix. After testing this PR with a multi-stage docker build I figured out that without these lines the installation of `postgres-client` failed. Maybe you should adjust the comment.